.github/workflows/auto_update_libs.yaml

```diff
@@ -0,0 +1,11 @@
+name: Auto-update charm libraries
```

Comment:
Instead of a cron job can we look into using renovate bot for auto update dependencies similar to https://github.com/jnsgruk/parca-k8s-operator?

Reply:
I'm a big advocate for adopting renovate to our repos and start pinning the dependencies so that we get repeatable builds, however, renovate does not support lib charm updates. This is one of the drawbacks of not using standard pip packages for those libraries.
We could of course fork renovate to add support for this, but it won't be a significant difference from a UX perspective and it would probably be higher maintenance.